Fixed Illumio Saas solutions workbook issue

v-prasadboke opened the pull request:
Required items, please complete
   
   Change(s):
   - Corrected workbook relative path in Workbooksmetadata

   Reason for Change(s):
   - Missing Playbook content from maintemplate 

   Version Updated:
   - no

   Testing Completed:
   - Yes
   - 
<img width="682" alt="image" src="https://github.com/user-attachments/assets/8182741c-4c8b-4a0d-b9bd-72b46de09a39">

v-prasadboke committed:
Update WorkbooksMetadata.json

v-prasadboke committed:
Solution packaged

ashwinvenkatesha commented:
Can we have a call regarding this?  @v-prasadboke 
Why does the screenshot refer to Illumio Core? That is our onprem connector, whereas the files being modified are for IllumioSaaS.
Also, I changed the version to 1.1.0 because I added a new workbook when compared to 1.0.0 beta version.

v-prasadboke commented:
> Can we have a call regarding this? @v-prasadboke Why does the screenshot refer to Illumio Core? That is our onprem connector, whereas the files being modified are for IllumioSaaS. Also, I changed the version to 1.1.0 because I added a new workbook when compared to 1.0.0 beta version.

Hello @ashwinvenkatesha, no worries about the Illumio core. It was just used as testing purpose.

v-prasadboke commented:
Issue resolved by author
